src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions_Parsing.cs

```diff
@@ -49,5 +50,54 @@ private static string GetExpressionBodyPreferenceEditorConfigString(CodeStyleOpt
                     throw new NotSupportedException();
             }
         }
+
+        private static CodeStyleOption<PreferBracesPreference> ParsePreferBracesPreference(
+            string optionString,
+            CodeStyleOption<PreferBracesPreference> defaultValue)
+        {
+            if (CodeStyleHelpers.TryGetCodeStyleValueAndOptionalNotification(
+                optionString,
+                out var value,
+                out var notificationOption))
```

Comment:
nested wrapping not being indented against its parent looks pretty strange.

Reply:
➡️ The current layout is what I expected